ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -36,32 +39,46 @@ component:
   guidevane: "Control vanes that meter refrigerant (on a centrifugal chiller) or air (on an AHU)."
   heater: "Component which provides heat to media."
   humidifier: "Component which humidifies."
+  interface: “A device/point which enables 2 systems/devices to communicate” # added 25/08/20
+  meter: "A device that measures and records the quantity, degree, or rate of something." # Added 10/09/20
+  module: "One of a set of separate parts that, when combined, form a complete whole device/system." # Added 05/10/20
   pump: "Component used for the distribution of liquid media."
+  rack: "Component used in comms rooms for mounting servers & switches" # Added 25/08/20
   shade: "Window covering."
-  tank : "Storage device for media."
+  tank: "Storage device for media."
   valve: "Component which meters the flow of water within a system or device."
   vane: "Component for guiding media flow."
   wheel: "Component used for transfer of heat from incoming to outgoing air streams. "
 descriptor: 
+  duty: "used to describe device used in duty/standby arrangement" # Added 27/08/20
+  standby: "used to describe device used in duty/standby arrangement or waiting and ready to start" # Added 27/08/20
+  runtime: "accumulation of time a device or system has been running (quite often used for maintenance/servicing purposes)" # Added 18/08/20
```

Comment:
trevor should probably weigh in on these.  the first two I'd have to see in context.  runtime seems like it is a measurement or a combination of a measurement and a measurement descriptor